src/lib/template-result.ts

```diff
@@ -20,6 +20,28 @@ import {reparentNodes} from './dom.js';
 import {TemplateProcessor} from './template-processor.js';
 import {boundAttributeSuffix, lastAttributeNameRegex, marker, nodeMarker} from './template.js';
 
+let policy: Pick<TrustedTypePolicy, 'createHTML'>|undefined;
+
+/**
+ * Turns the value to trusted HTML. If the application uses Trusted Types the
+ * value is transformed into TrustedHTML, which can be assigned to execution
+ * sink. If the application doesn't use Trusted Types, the return value is the
+ * same as the argument.
+ */
+function convertConstantTemplateStringToTrustedHTML(value: string): string|TrustedHTML {
+  // tslint:disable-next-line
+  const w = window as any
+  const TrustedTypes = (w.trustedTypes || w.TrustedTypes) as TrustedTypePolicyFactory;
+  if (TrustedTypes && !policy) {
```

Comment:
Can you move this outside the function too?

Reply:
Same as above